WooCommerce/Classes/View Modifiers/View+NoticesModifier.swift

```diff
@@ -34,6 +34,7 @@ struct NoticeModifier: ViewModifier {
 
     func body(content: Content) -> some View {
         content
+            .animation(nil, value: notice) // Only apply the animation to the notice stack, not the rest of the view content
             .overlay(buildNoticeStack())
             .animation(.easeInOut, value: notice)
     }
```

Comment:
Nice catch, what do you think about framing it like:
```swift
.overlay(
    buildNoticeStack()
        .animation(.easeInOut, value: notice)
)
```

That way it is more clear that the animation should only apply to what is inside the overlay.

Reply:
Nice! I didn't think of doing it like that, and it does make it clearer/simpler. Updated in [303bcae](https://github.com/woocommerce/woocommerce-ios/pull/6453/commits/303bcae886d875f4bc7c081e98daaa7d2c6dfc94).